src/pages/tiptap.tsx

```diff
@@ -0,0 +1,181 @@
+            <div>Test</div>
+            <div>Test</div>
+            <div>Test</div>
+            <div>Test</div>
+            <div>Test</div>
+            <div>Test</div>
+            <div>Test</div>
+            <div>Test</div>
+            <div>Test</div>
+            <div>Test</div>
+            <div>Test</div>
+            <div>Test</div>
+            <div>Test</div>
+            <div>Test</div>
+            <div>Test</div>
+            <div>Test</div>
+            <div>Test</div>
+            <div>Test</div>
+            <div>Test</div>
+            <div>Test</div>
+            <div>Test</div>
+            <div>Test</div>
+            <div>Test</div>
+            <div>Test</div>
+            <div>Test</div>
+            <div>Test</div>
+            <div>Test</div>
+
+
+          </div>
+          <div className="flex flex-col w-full h-full px-5 pt-5">
+            <div className="flex w-full h-auto border-x border-t border-black to bg-slate-200">
+              <button
+                onClick={() => {
+                  editor?.commands.toggleBold();
+                  editor?.commands.focus();
+                }}
+                className="w-10 h-10 font-bold hover:bg-slate-300 transition ease-in-out"
+              >
+                B
+              </button>
+              <div className="border-x border-black" />
+              <button
+                onClick={() => {
+                  editor?.commands.toggleItalic();
+                  editor?.commands.focus();
+                }}
+                className="w-10 h-10 italic hover:bg-slate-300 transition ease-in-out"
+              >
+                I
+              </button>
+              <div className="border-x border-black" />
+              <button
+                onClick={() => {
+                  editor?.commands.toggleStrike();
+                  editor?.commands.focus();
+                }}
+                className="w-10 h-10 hover:bg-slate-300 transition ease-in-out"
+              >
+                <s>S</s>
+              </button>
+              <div className="border-x border-black" />
+              <button
+                onClick={() => {
+                  editor?.commands.toggleBlockquote();
+                  editor?.commands.focus();
+                }}
+                className="w-10 h-10 hover:bg-slate-300 transition ease-in-out"
+              >
+                "
+              </button>
+              <div className="border-x border-black" />
+              <button
+                onClick={() => {
+                  editor?.commands.toggleCodeBlock();
+                  editor?.commands.focus();
+                }}
+                className="w-10 h-10 hover:bg-slate-300 transition ease-in-out"
+              >
+                &lt;
+              </button>
+              <div className="border-x border-black" />
+              <button
+                onClick={() => {
+                  editor?.commands.toggleBulletList();
```

Comment:
selecting multiline doesn't give me the whole div but this is my suggested edit for this section 

```    <div className="border-4 border-green-900 h-screen overflow-auto">
      <div className="flex justify-between items-center gap-5 px-20 py-2 border-b border-brown">
        <div className="flex items-center gap-5 text-">
          <input 
            type="text"
            placeholder="Untitled"
            className="w-60 h-10 bg-transparent border-transparent border-b-2 placeholder-green-900 text-green-900 hover:border-brown focus:border-brown transition ease-in-out focus:outline-none"
          />
          <button className="py-1 w-24 text-background-200 bg-green-700 rounded-sm">
            Save
          </button>
          <button className="py-1 w-24 text-background-200 bg-green-700 rounded-sm">
            Download
          </button>
          <button className="py-1 w-24 text-background-200 bg-green-700 rounded-sm">
            Publish
          </button>
        </div>
      </div>
      <div className="flex border-2 border-blue-500 h-screen">
        <div className="border-4 border-green-500 flex-1 pl-3 overflow-hidden hover:overflow-scroll">
          <div>Test</div>
          <div>Test</div>
          <div>Test</div>
          <div>Test</div>
          <div>Test</div>
          <div>Test</div>
          <div>Test</div>
          <div>Test</div>
          <div>Test</div>
          <div>Test</div>
          <div>Test</div>
          <div>Test</div>
          <div>Test</div>
          <div>Test</div>
          <div>Test</div>
          <div>Test</div>
          <div>Test</div>
          <div>Test</div>
          <div>Test</div>
          <div>Test</div>
          <div>Test</div>
          <div>Test</div>
          <div>Test</div>
          <div>Test</div>
          <div>Test</div>
          <div>Test</div>
          <div>Test</div>
          <div>Test</div>
          <div>Test</div>
          <div>Test</div>
          <div>Test</div>
          <div>Test</div>
          <div>Test</div>
          <div>Test</div>
          <div>Test</div>
          <div>Test</div>
          <div>Test</div>
          <div>Test</div>
          <div>Test</div>


        </div>
        <div className="flex flex-[4_1_0] flex-col border-2 border-red-500 p-5 overflow-auto">
          <div className="flex border-x border-t border-green-800 bg-background-400">
            <button
              onClick={() => {
                editor?.commands.toggleBold();
                editor?.commands.focus();
              }}
              className="w-10 h-10 font-bold text-green-900 hover:bg-background-600 transition ease-in-out"
            >
              B
            </button>
            <div className="border-x border-green-900" />
            <button
              onClick={() => {
                editor?.commands.toggleItalic();
                editor?.commands.focus();
              }}
              className="w-10 h-10 italic text-green-900 hover:bg-background-600 transition ease-in-out"
            >
              I
            </button>
            <div className="border-x border-green-900" />
            <button
              onClick={() => {
                editor?.commands.toggleStrike();
                editor?.commands.focus();
              }}
              className="w-10 h-10 text-green-900 hover:bg-background-600 transition ease-in-out"
            >
              <s>S</s>
            </button>
            <div className="border-x border-green-900" />
            <button
              onClick={() => {
                editor?.commands.toggleBlockquote();
                editor?.commands.focus();
              }}
              className="w-10 h-10 text-green-900 hover:bg-background-600 transition ease-in-out"
            >
              "
            </button>
            <div className="border-x border-green-900" />
            <button
              onClick={() => {
                editor?.commands.toggleCodeBlock();
                editor?.commands.focus();
              }}
              className="w-10 h-10 text-green-900 hover:bg-background-600 transition ease-in-out"
            >
              &lt;
            </button>
            <div className="border-x border-green-900" />
            <button
              onClick={() => {
                editor?.commands.toggleBulletList();
                editor?.commands.focus();
              }}
              className="w-10 h-10 text-green-900 hover:bg-background-600 transition ease-in-out"
            >
              •
            </button>
            <div className="border-x border-green-900" />
            <button
              onClick={() => {
                editor?.commands.setBlockquote();
                editor?.commands.focus();
              }}
              className="w-10 h-10 text-green-900 hover:bg-background-600 transition ease-in-out"
            >
              "
            </button>
            <div className="border-x border-green-900" />
            
            <div className="flex-1"></div>

            <div className="px-5 text-sm">
              <div>
                Character Count: {charCount}
              </div>
              <div>
                Word Count: {wordCount}
              </div>
            </div>
          </div>
          <div className="flex-1 bg-background-200 border border-green-900">
            <EditorContent editor={editor}/>
          </div>
        </div>
      </div>
    </div>
```